sphinx/source/model.rst

```diff
@@ -487,6 +487,28 @@ a built-in shader to create the Dissolve transform::
 Using the Model displayable as the child of a displayable is incompatible
 with :tpref:`mesh`, as the two both create models inside Ren'Py.
 
+Animated Shaders
+----------------
+
+When using shaders that depend on `u_time` to animate, one must be aware, that
```

Comment:
Oh, I forgot : in shinx you need double-backquotes for `fixed-width text`, not simple backquotes like in markdown. This needs to be \``u_time``, and so on.

Reply:
Right, single quotes are italics... Fixed.